account_invoice_import_invoice2data/README.rst

```diff
@@ -6,7 +6,7 @@
 Account Invoice Import Invoice2data
 ===================================
 
-This module is an extension of the module *account_invoice_import*: it adds support for regular PDF invoices i.e. PDF invoice that don't have an embedded XML file. It uses the `invoice2data library <https://github.com/m3nu/invoice2data>`_ which takes care of extracting the text of the PDF invoice, find an existing invoice template and execute the invoice template to extract the useful information from the invoice.
+This module is an extension of the module *account_invoice_import*: it adds support for regular PDF invoices i.e. PDF invoice that don't have an embedded XML file. It uses the `invoice2data library <https://github.com/m3nu/invoice2data>`_ which takes care of extracting the text of the PDF invoice, find an existing invoice template and execute the invoice template to extract the useful information from the invoice. This module has been extended with ``account_invoice_import_invoice2data_templates`` so that import templates can also be managed from the Odoo environment.
```

Comment:
Sounds better to say `Invoices can be stored on the filesystem or in the database` than`has been extended with account_invoice_import_invoice2data_templates` (sounds like a changelog)